src/footer/footer.tsx

```diff
@@ -16,7 +16,7 @@ const FooterComponent = (): JSX.Element => {
 
   return (
     <Footer
-      title={t('footer.text', 'Footer lorem ipsum')}
+      title={t('footer.text', 'Plot rental')}
```

Comment:
Q: Is it leasing or renting?

Reply:
This follows the mvj-ui-public convention renaming Plot search as Plot rental.